kptncook/mealie.py

```diff
@@ -32,70 +41,71 @@ class RecipeIngredient(BaseModel):
     note: str | None
     unit: UnitFoodBase | None
     food: UnitFoodBase | None
-    disable_amount: bool = True
-    quantity: float = 1
+    disableAmount: bool = True
+    quantity: float | None
 
 
 class RecipeSummary(BaseModel):
     id: UUID4 | None
 
-    user_id: UUID4 | None = Field(None, alias="userId")
-    group_id: UUID4 | None = Field(None, alias="groupId")
+    userId: UUID4 | None = Field(None, alias="userId")
+    groupId: UUID4 | None = Field(None, alias="groupId")
 
     name: str | None
     slug: str = ""
     image: Any | None
-    recipe_yield: str | None
+    recipeYield: str | None
 
-    total_time: str | None = None
-    prep_time: str | None = None
-    cook_time: str | None = None
-    perform_time: str | None = None
+    totalTime: str | None = None
+    prepTime: str | None = None
+    cookTime: str | None = None
+    performTime: str | None = None
 
     description: str | None = ""
-    recipe_category: list[str] | None = []
-    # tags: list[RecipeTag] | None = []
-    tags: list[RecipeTag | str] | None = []
+    recipeCategory: list[str] | None = []
```

Comment:
This will throw an exceptions if there are any Categories assigned to a recipe. Change to `list[RecipeCategory]` should do the trick, but I haven't tested that yet.

Reply:
Ok, let's deal with the bug when it shows up 😃